src/components/InterestSignupForm.js

```diff
@@ -33,11 +36,27 @@ class InterestSignupForm extends Component {
     render () {
         let canSubmit = (this.state.email.length > 3);
         return (
-            <div>
-                <input name="email" type="email" value={this.state.email} onChange={this.changeEmail.bind(this)} placeholder="Email"/>
-                <button type="submit" onClick={this.handleSubmit.bind(this)} disabled={!canSubmit}>Submit</button>
-                <p>{this.state.message}</p>
-            </div>);
+          <div className="con">
+            <div className="flexchild">
+              <h1 className="pink">BoilerMake</h1>
+              <h3 className="pink">Purdue University</h3>
+              <h3 className="lightBlue">SEPT 29  - OCT 1 2017</h3>
+              <br/>
+              <p>Sign up for updates on when applications open!</p>
+
+              <div className="interestForm">
+
+                  <input name="email" type="email" value={this.state.email} onChange={this.changeEmail.bind(this)} placeholder="Email"/>
```

Comment:
inputs need labels for accessibility
Also, focus causes a weird effect:
<img width="910" alt="screen shot 2017-06-19 at 3 49 32 pm" src="https://user-images.githubusercontent.com/1365665/27303002-38cabd2c-5507-11e7-92f9-7a96e1f5ddf6.png">